.ci.yaml

```diff
@@ -40,6 +42,8 @@ targets:
     recipe: cocoon/device_doctor
     properties:
       add_recipes_cq: "true"
+      script: "build.sh"
```

Comment:
It would help to add the full path here (e.g. `device_doctor/tools/build.sh`). This will reduce the hard coding in the recipe.

Similar to below, you may want to add the full cipd name (like flutter/device_doctor/mac-amd64)

Reply:
Sure, updated. 

Yeah in recipe I would need to extract information such as `device_doctor` from the full path, and concatenate it with `.zip` etc. I am planning to extract platform information from full path as well, so that I can compare with the three character platform name ('win', 'mac' and 'lin') from api.platform.name in recipe

Reply:
Sounds good to me, feel free to remove the platform from the end of cipd_name (up to you, but I don't think it'll change often). Might be good for cases where the dimensions aren't correctly assigned, and a mac arm is assigned instead of a mac intel